core/store/src/adapter/trie_store.rs

```diff
@@ -156,14 +150,145 @@ impl<'a> TrieStoreUpdateAdapter<'a> {
         )
     }
 
+    /// Set the mapping from `child_shard_uid` to `parent_shard_uid`.
+    /// Used by Resharding V3 for State mapping.
+    #[cfg(test)]
```

Comment:
Eventually this will be called from the resharding trigger right?

Reply:
Yes